src/build-time-render/BuildTimeRender.ts

```diff
@@ -124,9 +141,19 @@ export default class BuildTimeRender {
 			blockScripts.forEach((blockScript, i) => {
 				html = html.replace(
 					'</body>',
-					`<script type="text/javascript" src="${scriptPrefix}${blockScript}" async="true"></script></body>`
+					`<script type="text/javascript" src="${scriptPrefix}${blockScript}"></script></body>`
 				);
 			});
+			additionalScripts
+				.sort((script1, script2) => {
```

Comment:
This `sort` forces `main.` scripts to be loaded last. If `main.12345.bundle.js` is loaded and executed before the additional scripts, the additional scripts have a chance to be loaded by the main bundle anyways, thus negating all this preloading.

Reply:
can we not make this an exact match? otherwise it would pick up say `main.container.js`?